install/macos/build-app.py

```diff
@@ -75,9 +75,7 @@ def sign_app_bundle(build_path, dist_path, app_path):
     icon_path = os.path.join(root, "install", "macos", "dangerzone.icns")
 
     print("○ Code signing app bundle")
-    identity_name_application = (
-        "Apple Distribution: Freedom of the Press Foundation (94ZZGGGJ3W)"
-    )
+    identity_name_application = "Apple Development: Trevor Timm (TW4JM8ZCU)"
```

Comment:
Let's also change a similar line in `RELEASE.md`:

https://github.com/freedomofpress/dangerzone/blob/1c0dfb45f5dc6cc36fbfa2983ad2e1ff45aaef91/RELEASE.md?plain=1#L169

Reply:
Speaking of this change, it may be worth remove this line before it:

https://github.com/freedomofpress/dangerzone/blob/1c0dfb45f5dc6cc36fbfa2983ad2e1ff45aaef91/RELEASE.md?plain=1#L168

This version is 3 major versions behind the current one. At this point, I don't think any developer will have a problem.

Reply:
Actually, I did the above already, since I found some other small issues as well.